pkg/server/plugin/upstreamca/awssecret/awslib_fake.go

```diff
@@ -0,0 +1,61 @@
+package awssecret
+
+import (
+	"io/ioutil"
+	"net/http"
+
+	"github.com/aws/aws-sdk-go/aws/request"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/aws/aws-sdk-go/service/secretsmanager/secretsmanageriface"
+	"github.com/stretchr/testify/suite"
+)
+
+type fakeSecretsManagerClient struct {
+	suite.Suite
```

Comment:
the test suite should be its own independent struct that lives in the `_test.go` file.